src/ForeElementMixin.js

```diff
@@ -6,272 +6,284 @@ import {
   evaluateXPathToString,
 } from './xpath-evaluation.js';
 import getInScopeContext from './getInScopeContext.js';
-import {Fore} from './fore.js';
+import { Fore } from './fore.js';
 
 /**
  * Mixin containing all general functions that are shared by all Fore element classes.
- * @param superclass
- * @returns {{readonly properties: {ref: {type: StringConstructor}, context: {type: ObjectConstructor}, nodeset: {type: ObjectConstructor}, model: {type: ObjectConstructor}, inScopeVariables: {type: MapConstructor}, modelItem: {type: ObjectConstructor}}, new(): ForeElementMixin, context: null, model: null, modelItem: {}, ref: *|string, inScopeVariables: null, nodeset: *, prototype: ForeElementMixin}}
+ * @extends {HTMLElement}
```

Comment:
@JoernT , this one is ionteresting: the mixin part of this mixin was not actually used! This was only used as plain inheritance. Actually coding it as such adds a lot of nice autocompletes!